### DISCUSSION TITLE
I need the GitHub support to remove 2FA from the account.

### DISCUSSION THREAD

**Discussion by sysop-88** [community/community#127780] [Other Feature Feedback, Questions, & Ideas]
I lost my **2FA recovery** for my account. I changed my computer. I need **GitHub support** to remove **2FA** because it is locked out. I made a support ticket [2fa password recovery #2818086](https://support.github.com/ticket/personal/0/2818086) I need to remove 2FA to log in since I lost my 2FA recovery.

I also find many discussion on GitHub community about the support tickets have not been replied for long time. Is there anyway I can contact the GitHub support team about the support ticket?

**Comment by CodeModeYT** [comment]
Normally, when adding 2fa, you also add a backup method, like SMS. Have you done that?

Also, you should check if you still have your GitHub account recovery codes stored somewhere.

Apart from that, I can only recommend reading this article: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials

PS: my personal experience with GitHub support is the exact opposite, I got my answer after less than 24h, it's deifnetly worth a try!

**Comment by ebndev** [comment]
Hi @sysop-88,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
